diagnostic.md

```diff
@@ -150,6 +183,12 @@ div {
 ```
 
 <!-- your answer starts here -->
+div one - float right (stated)
+div two - float left (from div)
+div three - float left (from div)
+dive four - new line (no float)
+div five - float left (from div)
+
```

Comment:
```.divOne``` will be in the top-right corner;
```.divTwo``` will be in the top-left corner, with .divThree immediately to its right;
```.divFour``` will be underneath ```.divTwo```, and ```.divFive``` will sit immediately to the right of ```.divFour``` (under ```.divThree```).